sim-cli/src/parsing.rs

```diff
@@ -140,23 +188,94 @@ impl TryFrom<&Cli> for SimulationCfg {
     }
 }
 
+struct NodeMapping(HashMap<PublicKey, NodeInfo>, HashMap<String, NodeInfo>);
```

Comment:
let's name these fields? Original motivation to ask for the struct was to make this more easily readable